plugins/module_utils/ilo_redfish_utils.py

```diff
@@ -231,3 +608,1407 @@ def set_wins_registration(self, mgrattr):
         if not response['ret']:
             return response
         return {'ret': True, 'changed': True, 'msg': "Modified %s" % mgrattr['mgr_attr_name']}
+
+    def verify_drive_count(self, raid_details, logical_drives_count):
+        if len(raid_details) != logical_drives_count:
+            return {
+                "ret": False,
+                "changed": False,
+                "msg": "Logical drive count in raid details is not matching with logical drives present in the server"
+            }
+        return {
+            "ret": True,
+            "msg": "Drive count is same as input"
+        }
+
+    def verify_logical_drives(self, raid_details, check_length=False):
```

Comment:
Is this going to remain specific to Smart Storage?

Reply:
Yes, this retrieves information about logical drives from the SmartStorage space.

Reply:
Resolving this conversation. Thanks!